app.js

```diff
@@ -1,5 +1,77 @@
 
 // state
+function getDefaultVolume() {
+    
+    const defaultVolume = {
+        level: 10,
+    };
+    return defaultVolume;
+    
+}
+
+let volume = getDefaultVolume();
+let click = false;
+
+
+
+//variables
+const controlsSection = document.getElementById('controls-section');
+
+const [downButton, upButton] = controlsSection.querySelectorAll('button');
+let volumeDisplay = controlsSection.querySelector('span');
+console.log(volumeDisplay);
+
+//event listeners
+
+downButton.addEventListener('click', () => {
+    
+    downLevel();
+    levelDisplay();
+    volumeDisplay.textContent = volume.value;
```

Comment:
This line should be in your `levelDisplay` function

Reply:
But if you had done: `volumeDisplay.textContent = volume.level` I think it would have all worked;